cegs_portal/search/templates/search/v1/experiment_list.html

```diff
@@ -63,6 +62,7 @@
 
 {% block content %}
 <div class="min-w-full">
+    <div class="title-separator"></div>
```

Comment:
I don't think these changes are reflected in the screenshot.

Reply:
Added screenshot